### PR TITLE
Improve performance of `.lvCompare()`

### PR DESCRIPTION
**Pull request by Qile0317**
A general improvement on the runtime of `combineBCR()` by accelerating `.lvCompare()` with rcpp by a small constant factor.

### CODE REVIEW

**Comment by Qile0317** [comment]
@ncborcherding Just realized that the vignette build failed in the R-CMD-check with the following:
```
Error: Error: processing vignette 'vignette.Rmd' failed with diagnostics:
object 'uni_IG' not found
--- failed re-building ‘vignette.Rmd’
```
and also
```
The magick package is required to crop "/tmp/Rtmp7YAETR/Rbuild179452a8e5c7/scRepertoire/vignettes/vignette_files/figure-html/unnamed-chunk-55-1.png" but not available.
Quitting from lines 1146-1159 [unnamed-chunk-57] (vignette.Rmd)
```
Will get on that later. Unsure why vignette is failing since it wasn't really touched.

File: R/clonalCluster.R [hunk 1]
```diff
@@ -161,9 +161,9 @@ clonalCluster <- function(input.data,
                         }
                         colnames(output.list[[x]]) <- c(paste0(chain, "_cluster"), ref2)
                         output.list[[x]]
-                        
+
   })
-  cluster <- bind_rows(cluster.list)
+  cluster <- bind_rows(cluster.list) # the TRA_cluster isnt assigned in the failing test
```
Comment (Qile0317) [inline R/clonalCluster.R:166]: @ncborcherding Could you maybe describe what exactly this intermediate variable is supposed to be? Struggling to debug downstream atm.

Comment (ncborcherding) [reply R/clonalCluster.R:166]: @Qile0317 Here the ```cluster.list``` is the cluster assignments from the ```.lv.compare()``` function that are grouped into list elements, ```bind_rows()``` is just forming a data frame from the list.

**Review by ncborcherding — APPROVED** [review]

**Comment by Qile0317** [comment]
@ncborcherding actually - I just commented out a failing test - I think this is still broken :P this was my fault for not communicating well - I can do another PR for undoing the merge?

**Comment by ncborcherding** [comment]
@Qile0317 Don't worry about it and sorry for not catching it - why don't you make a new PR for the sole issue itself. I think that will save both of use time and be more straightforward. 

Nick

**Pull request merged**